ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/NodeCve/NodeCvePage.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { PageSection, Breadcrumb, Divider, BreadcrumbItem } from '@patternfly/react-core';
+import { useParams } from 'react-router-dom';
+
+import PageTitle from 'Components/PageTitle';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import { getOverviewCvesPath } from '../utils/searchUtils';
+
+const workloadCveOverviewCvePath = getOverviewCvesPath({
+    entityTab: 'CVE',
+});
+
+function NodeCvePage() {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { cveId } = useParams() as { cveId: string };
+
+    const nodeCveName = cveId; // TODO Replace me with queried data
+
+    return (
+        <>
+            <PageTitle title={`Node CVEs - NodeCVE ${nodeCveName}`} />
```

Comment:
Would it be useful to create a function that generates these page titles? We can reuse them between Workload CVEs, Node CVEs, and Platform CVEs. I think you might have done something similar for Collections

Reply:
This is a great idea, now with Node and Platform we are going to be duplicating this code quite a few times. I'll add this in a follow up.